features/adres.feature

```diff
@@ -3,9 +3,13 @@
 @post-assert
 Functionaliteit: Adresvelden vullen
 
-  Rule: Voor een binnenlands adres wordt veld "straat" gevuld met de naam openbare ruimte (11.15) wanneer die bekend is, en anders met straatnaam (11.10). Veld "korteNaam" wordt gevuld met straatnaam (11.10).
 
-    Abstract Scenario: opnemen straat bij een binnenlands adres
+# Onderstaande rule en scenario moeten m.i. nog steeds verplaatst worden naar 
+  Rule: Voor een binnenlands adres wordt:
+          -  veld "officieleStraatnaam" gevuld met de naam openbare ruimte (11.15). 
+          -  veld "korteStraatnaam" wordt gevuld met straatnaam (11.10).
+
+    Abstract Scenario: opnemen officieleStraatnaam en korteStraatnaam bij een binnenlands adres
```

Comment:
2 vragen:
- conflicteert dit niet met p.r. 1505? Daar is deze scenario verwijderd
- wat beschrijft deze scenario dat al niet is beschreven met de scenario's in features/bevragen/adres/verblijfadres/fields.feature?

Reply:
met @MelvLee eens, deze moet verwijderd worden

Reply:
van mij mag je de rule en bijbehorende scenario verwijderen

Reply:
Verwijderd.